chore: Improve our user documentation (#733)
Unleash/unleash: docs/user_guide/activation-strategies.md

**Parameters**

- **stickiness** is used to define how we guarantee consistency for gradual rollout. The same userId and the same rollout percentage should give predictable results. Configuration that should be supported:
  - **default** - Unleash chooses the first value present on the context in defined order userId, sessionId, random.
  - **userId** - guaranteed to be sticky on userId. If userId not present the behavior would be false
  - **sessionId** - guaranteed to be sticky on sessionId. If sessionId not present the behavior would be false.
  - **random** - no stickiness guaranteed. For every isEnabled call it will yield a random true/false based on the selected rollout percentage.
- **groupId** is used to ensure that different toggles will **hash differently** for the same user. The groupId defaults to _feature toggle name_, but the use can override it to _correlate rollout_ of multiple feature toggles.
- **rollout** The percentage (0-100) you want to enable the feature toggle for.

### Customize stickiness (beta)

> By enabling the stickiness option on a custom context field you can use it together with the flexible rollout strategy. This will guarantee a consistent behavior for specific values of this context field. PS! support for this feature currently being supported by the following SDKs:

Reviewer: ```suggestion
By enabling the stickiness option on a custom context field you can use it together with the flexible rollout strategy. This will guarantee a consistent behavior for specific values of this context field.
NB! this feature is currently only supported by the following SDKs:
```
